src/components/error/ApplicationError.jsx

```diff
@@ -0,0 +1,49 @@
+const ApplicationError = () => {
+  const onRefresh = (event) => {
+    event.preventDefault();
+    location.reload();
+  };
+
+  const onGoBack = (event) => {
+    event.preventDefault();
+    window.location.replace(document.referrer);
+  };
+
+  return (
+    <div>
+      <h1 className="govuk-heading-l">
+        Sorry, there is a problem with the service
+      </h1>
+      <div
+        data-testid="app-error-message"
+        className="govuk-error-summary__body"
+      >
+        Any unsaved changes will have been lost. Please try again later. <br />
```

Comment:
Could you wrap these lines in `<p>` tags to avoid the `<br/>`?

Reply:
done